src/pipeline/client.rs

```diff
@@ -219,23 +217,18 @@ where
                     span: _span,
                     res,
                 })) => {
-                    #[cfg(feature = "tracing")]
                     let guard = _span.enter();
-                    #[cfg(feature = "tracing")]
                     tracing::event!(Level::TRACE, "request received by worker; sending to Sink");
```

Comment:
could be more concise as
```suggestion
                    tracing::trace!("request received by worker; sending to Sink");
```